qt/widgets/common/src/FitPropertyBrowser.cpp

```diff
@@ -1042,7 +1042,6 @@ std::string FitPropertyBrowser::defaultPeakType() {
 void FitPropertyBrowser::setDefaultPeakType(const std::string &fnType) {
```

Comment:
Should we still refer to this a the default peak type or current peak type here?

Reply:
Yes, that would be useful.

Reply:
I'll leave the name for now because there are a few other "default" variables in the base class, e.g. the background type. And it is at least a default in the context of the fit browser itself.